runtime/service.go

```diff
@@ -454,10 +456,19 @@ func (s *service) CreateVM(requestCtx context.Context, request *proto.CreateVMRe
 	)
 
 	s.vmStartOnce.Do(func() {
-		err = s.createVM(ctxWithTimeout, request)
+		if err = s.createVM(ctxWithTimeout, request); err != nil {
```

Comment:
Could you add some tests to validate the response?

Reply:
I did a small refactor to make the construction of the response a separate function in order to make the test easier, but then realized that constructing the response is just copying values over from the `service` struct. 

What would you like to see asserted?

Reply:
How about checking VMID in https://github.com/firecracker-microvm/firecracker-containerd/blob/master/runtime/service_integ_test.go#L1792?

Reply:
@kzys Done. Getting CI green, will ping when done